bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
-    pplacer_options.add_argument('--no_merge_reads', action="store_true", help='Merge forward and reverse read alignments for placement (default=False)', default=False)
+    pplacer_options.add_argument('--placements_cutoff', metavar='confidence', help='This flag allows you to change the likelihood cutoff for phylogenetic placement of reads (default=0.75).', default=0.75)
+    pplacer_options.add_argument('--resolve_placements', action="store_true", help='Ignore the placements cutoff and simply use the best placement assigned to the read. (default=False)', default=False)
+    pplacer_options.add_argument('--no_merge_reads', action="store_true", help='When this flag is specified, The alignment of the forward and reverse reads will be merged before placement. This provides more information for phylogenetic placement, and the classifications provided by pplacer generally improve, so try to use this flag when you can. (default=False)', default=False)
 
     nucleotide_options = graft_parser.add_argument_group('nucleotide search-specific options')
-    nucleotide_options.add_argument('--search_hmm_files', nargs='+', help='Specify .hmm files to use in search step', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--search_hmm_list_file', metavar='newline-separated file with paths to each search HMM', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--search_diamond_file', nargs=1, help='Specify .dmnd file to use in search step', default=None)
-    nucleotide_options.add_argument('--aln_hmm_file', help='Specify a single .hmm file to use in align step (default: the search_hmm_file if there is only 1)', default=argparse.SUPPRESS)
-    nucleotide_options.add_argument('--euk_hmm_file', help='Specify the .hmm file to use in the check for euk contamination', default=argparse.SUPPRESS)
+    nucleotide_options.add_argument('--search_hmm_files', nargs='+', help='This flag is to specify the HMM(s) used to search the data with - note you cannot change the HMM that is used to align the sequences. This is because alignments must match the alignment length of the sequences used in the placement tree, otherwise the placement step will fail. If providing more than one HMM, separate with a space.', default=argparse.SUPPRESS)
+    nucleotide_options.add_argument('--search_hmm_list_file', metavar='If you have many hmms you would wish to search with, put the path to each HMM file in a file, one per line and provide it to this flag.', default=argparse.SUPPRESS)
+    nucleotide_options.add_argument('--search_diamond_file', nargs=1, help='Specify a DIAMOND database with which to search/classify the reads.', default=None)
+    nucleotide_options.add_argument('--aln_hmm_file', help='Reads will be aligned to this HMM after identification. This option can only be used if no placement is required.', default=argparse.SUPPRESS)
```

Comment:
As in not "translation"? If so I agree. Also I notice that `--search_hmm_list_file` seems to be using metavar when it means help. Also, shouldn't these options be in a general search section rather than under nucleotides?